src/Controller/CommentController.php

```diff
@@ -100,11 +100,28 @@ public function comment(
             $em->persist($reply);
             $em->flush();
 
+            /*
```

Comment:
We have history with GIT, not sure we need to leave commented code in the codebase